tests/test_preprocess.py

```diff
@@ -24,14 +24,6 @@ def test01CropPad(self):
         vols_f = crop_pad(fftshift(fftn(vols[:, :, :, 0])), 8)
         self.assertTrue(np.allclose(results, vols_f, atol=1e-7))
 
-    def test02Downsample(self):
```

Comment:
Maybe we should rename the remaining tests since they are named `test01...`, `test02...`, ...

Reply:
better yet, no numbers!